iceoryx_posh/test/integrationtests/test_client_server.cpp

```diff
@@ -82,6 +95,25 @@ class ClientServer_test : public RouDi_GTest
 };
 constexpr iox::units::Duration ClientServer_test::DEADLOCK_TIMEOUT;
 
+class BigPayloadClientServer_test : public ClientServer_test
+{
+  public:
+    BigPayloadClientServer_test()
+        : ClientServer_test(
+            MinimalRouDiConfigBuilder().payloadChunkSize(BIG_PAYLOAD_SIZE + 128).payloadChunkCount(2).create())
```

Comment:
No, this is just an arbitrary number to make sure the payload is larger than 4GB.

Reply:
Btw, could you increase the size of the payload by at least the step size in the for loop. This is just to ensure to write beyond the 4GB mark.

Reply:
> No, this is just an arbitrary number to make sure the payload is larger than 4GB.

Actually this number is included in the constant `BIG_PAYLOAD_SIZE`
This might have been an arbitrary offset that should manage to fit the header